### PR TITLE
Use Node.js 20

### PR DESCRIPTION
**Pull request by LitoMore**
### Changes

- Upgrade GitHub Actions runtime to Node.js 20
- Upgrade labeler runtime to Node.js 20
- Use latest npm lockfile version

Files are auto-formatted by Prettier.

I intentionally didn't update `dist`. We could update `dist` on the version release rather than on every commit/PR.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -1,18 +1,18 @@
-name: 'Labeler'
-description: 'Automatically label new pull requests based on the paths of files being changed'
-author: 'GitHub'
+name: "Labeler"
```
Comment (mondeja) [inline action.yml]: Would you mind to add the same [Prettier config](https://github.com/simple-icons/simple-icons/blob/develop/.prettierrc.json) of all repos? That will use single quotes in formatting.

**Review by mondeja — CHANGES_REQUESTED** [review]
> I intentionally didn't update dist. We could update dist on the version release rather than on every commit/PR.

I think that is better to update on pre-commit with Husky and add the *dist/* folder to *.gitattributes* with `-diff` option because in the future we'll forget about that.

File: action.yml (hunk 1 above)
Comment (LitoMore) [reply action.yml]: np. I've updated all files with single quote.

**Comment by LitoMore** [comment]
> I think that is better to update on pre-commit with Husky and add the dist/ folder to .gitattributes with -diff option because in the future we'll forget about that.

We can do it on another PR.

In the meantime, we can add an action to check if we forgot to update the dist.

**Review by mondeja — APPROVED** [review]
> In the meantime, we can add an action to check if we forgot to update the dist.

This seems to be much more work. No problem, I'll add it in another PR.

**Pull request merged**